hollow/src/main/java/com/netflix/hollow/api/codegen/HollowConsumerJavaFileGenerator.java

```diff
@@ -82,4 +137,17 @@ private String createFullPackageName(String packageName, String subPackageName,
     private boolean isEmpty(String value) {
         return value == null || value.trim().isEmpty();
     }
+
+    private void appendImportFromBasePackage(StringBuilder builder, String leaf) {
+        builder.append("import ").append(packageName).append(".").append(leaf)
+            .append(";\n");
+    }
+
+    private void addToSetIfNotPrimitive(Set<String> set, String... schemaNames) {
```

Comment:
Minor since this is a small method but it might be better to rename "set" to importSet or something more descriptive ;)

Reply:
Can't hurt, will do

Reply:
Actually, I am not clear on this method.  is it supposed to convert schemaName into Hollow Package Import String?